src/components/SearchResults.vue

```diff
@@ -13,14 +13,19 @@
                 </div>
               </div>
            <div class="headline">
-             <span class="numbersFound">{{ resultHits || "0" }}</span> matches was found in
-               <span class="numbersFound"> {{ resultMatches || "0" }}</span>
-               <span v-if="resultHits > 1 || resultHits === null || resultHits === 0"> pdfs</span>
-               <span v-if="resultHits === 1"> pdf</span>
-           </div>
-           
-
-              <component :is="searchResultComponentName"  v-for="(item, index) in this.results" :result="item" :indexNumber="index" :queryString="queryDisplay" :key="index"  ></component>
+             <span class="numbersFound">{{ resultHits || "0" }}</span> matches was found.
+             <span v-if="loading === false && this.results.length > 0">Showing hit 
+              <span class="numbersFound"> {{ this.solrOptions.currentOffset + 1 }} - {{ this.solrOptions.currentOffset + this.solrOptions.shownResultsNumber > resultHits ? resultHits : this.solrOptions.shownResultsNumber + this.solrOptions.currentOffset }}</span>.</span>
+          </div>
+          <div v-if="loading === false && this.results.length > 0" class="pagingButtonContainer">
+            <button class="pagingButton back" :disabled="this.solrOptions.currentOffset <= 0" v-on:click="previousResults">previous 10</button>
+            <button class="pagingButton forward" :disabled="this.solrOptions.currentOffset + this.solrOptions.shownResultsNumber >= this.resultHits" v-on:click="nextResults">next 10</button>
+          </div>
+          <component :is="searchResultComponentName"  v-for="(item, index) in this.results" :result="item" :indexNumber="index" :queryString="queryDisplay" :key="index"  ></component>
+          <div v-if="loading === false && this.results.length > 0" class="pagingButtonContainer pageEnd">
+            <button class="pagingButton back" :disabled="this.solrOptions.currentOffset <= 0" v-on:click="previousResults">previous 10</button>
+            <button class="pagingButton forward" :disabled="this.solrOptions.currentOffset + this.solrOptions.shownResultsNumber >= this.resultHits" v-on:click="nextResults">next 10</button>
```

Comment:
With this simple pager it's ok to keep it here - if we do anything more fancy with clickable page numbers etc. we will move it to seperate component.

Reply:
Allrighty, i'll leave it for now.